package.json

```diff
@@ -0,0 +1,17 @@
+{
+  "name": "actuatejs",
+  "version": "1.0.0",
+  "description": "One line easy actuation of CSS animation sequences.",
+  "main": "index.js",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/lukejacksonn/Actuate.git"
+  },
+  "keywords": ["css", "animate", "animation", "sequencing", "promise"],
+  "author": "Luke Jackson <lukejacksonn@gmail.com>",
+  "license": "MIT",
+  "bugs": {
+    "url": "https://github.com/lukejacksonn/Actuate/issues"
+  },
+  "homepage": "https://github.com/lukejacksonn/Actuate#readme"
```

Comment:
it would yes, so the publish hook would generate the file on`npm publish` which means it would publish it, but would not add it to the repo. You would refernce the min file in the `package.json` and have the min file in the .gitignore :), yep you could potentially run that in a build or the publish script as well. maening people who just want to `<script src="//`> wouldnt needed it